src/subdomains/supporting/payment/services/fee.service.ts

```diff
@@ -260,16 +264,42 @@ export class FeeService {
         baseFee.payoutRefBonus &&
         (discountFee?.payoutRefBonus ?? true) &&
         additiveFees.every((fee) => fee.payoutRefBonus),
-      blockchain: Math.max(
-        blockchainFee *
-          (baseFee.blockchainFactor -
-            (discountFee?.blockchainFactor ?? 0) +
-            Util.sumObjValue(additiveFees, 'blockchainFactor')),
-        0,
+      blockchain: Math.min(
+        Math.max(
+          blockchainFee *
+            (baseFee.blockchainFactor -
+              (discountFee?.blockchainFactor ?? 0) +
+              Util.sumObjValue(additiveFees, 'blockchainFactor')),
+          0,
+        ),
+        Config.maxBlockchainFee,
       ),
     };
   }
 
+  private async getBlockchainFee(active: Active): Promise<number> {
+    if (isAsset(active)) {
+      const fee = await this.blockchainFeeRepo.findOneBy({
+        asset: { id: active.id },
+        updated: MoreThan(Util.minutesBefore(70)),
+      });
+
+      return fee ? fee.amount : this.getBlockchainMaxFee(active.blockchain);
+    } else {
+      return 0;
+    }
+  }
+
+  private async getBlockchainMaxFee(blockchain: Blockchain): Promise<number> {
+    const { fee } = await this.blockchainFeeRepo
+      .createQueryBuilder('fee')
+      .select('MAX(amount)', 'fee')
+      .innerJoin('fee.asset', 'asset')
+      .where('asset.blockchain = :blockchain', { blockchain })
+      .getRawOne<{ fee: number }>();
+    return fee;
```

Comment:
Filter for update more than 70 minutes before and return 0 as fallback `return fee ?? 0;`